src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,47 @@
+package duke;
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents a deadline task that user creates
+ */
+public class Deadline extends Task {
+    protected LocalDate by;
+    protected LocalTime time;
+
+    public Deadline(String description, LocalDate by, LocalTime time) {
+        super(description);
+        this.by = by;
+        this.time = time;
+    }
+
+    /**
+     * Return the string representation of deadline in the right date and time format
+     *
+     * @return a string with the right date and time format
+     */
```

Comment:
Perhaps you can add punctuation to the end of Javadoc descriptions and parameters to comply to the coding standard?